Added the doc for config import-* commands (#15)
hazelcast/clc-docs: docs/modules/ROOT/pages/clc-config.adoc

clc config add [target] [source] [flags] [key=value...]
----

You can pass key=value pairs to add configuration items.

.Configuration keys
[%collapsible]
====
include::partial$configuration-items.adoc[]
====

.Global parameters
[%collapsible]
====
include::partial$global-parameters.adoc[]
====

Example output:

[source,bash]
----
clc config add dev cluster.name=dev cluster.address=localhost:5701
----

== clc config import-cloud

Imports configuration from a {hazelcast-cloud} CLI sample.

> In order to get the URL for the {hazelcast-cloud}, follow the steps below:

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021****

> 1. On {hazelcast-cloud} console, visit:

Reviewer: ```suggestion
1. Open the {hazelcast-cloud} console, and select *Dashboard* -> *Connect Client* -> *CLI*
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**

> +

Reviewer: ```suggestion

```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**

> 	Dashboard -> Connect Client -> CLI

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**

> 2. Copy the URL in box 2 and pass it as the second parameter.

Reviewer: ```suggestion
2. Copy the URL from the second step of the on-screen instructions, and pass it as the second parameter.
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**

> Make sure the text is quoted before running:

Reviewer: ```suggestion
Ensure that the URL is enclosed in quotation marks, then run:
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021

+
[source,bash]
----

> clc config import-cloud my-config "https://api..."

Reviewer: would it be useful to include a suggestion rather than the ellipses? Something like:

clc config import-cloud my config "_URL copied from console_"

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**

----

.Global parameters
[%collapsible]
====
include::partial$global-parameters.adoc[]
====

== clc config import-java

This command imports a subset of Hazelcast Java client configuration and saves it as CLC configuration.

> Java keystore and truststore referred in the XML configuration are converted to PEM files.

Reviewer: ```suggestion
The Java keystore and truststore, referred in the XML configuration, are converted to PEM files.
```

Usage:

[source,bash]
----
clc config import-java [target] [source] [flags] [key=value...]
----

> You can pass `key=value` pairs to override configuration items.

Reviewer: ```suggestion
You can pass `key=value` pairs to override configuration options.
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021

.Configuration keys
[%collapsible]
====
include::partial$configuration-items.adoc[]
====

Parameters:

[cols="1m,1m,2a,1m"]
|===
|Parameter|Required|Description|Default

|`target`
|Required

> |Name or path to the configuration to be created

Reviewer: ```suggestion
|Name or path to the configuration to create
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021

|

| `source`
|Required
|Path to the Java client XML configuration
|

|`--ca-alias`
|Optional
|Alias of the CA certificate in the Java truststore
|

|`--cert-alias`
|Optional
|Alias of the certificate in the Java truststore
|

|`--key-alias`
|Optional
|Alias of the key in the Java keystore
|

|===

## Limitations

> 

Reviewer: ```suggestion

The following are not supported:

```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021

> * Only PKCS12 keystores/truststores are supported, not JKS

Reviewer: ```suggestion
* JKS. Only PKCS12 keystores/truststores are supported
```

Author: I meant JKS type keystores are not supported.

Author: I've updated that section to:
```
=== Limitations

Only PKCS12 keystores/truststores are supported.

The following are not supported:

* JKS keystores/trustores
* <import> and <config-replacers> tags
* Variables in .xml configuration
```
Would that be OK?

Reviewer: Even better than my suggestion! Thank you.

Author: Thanks!

> * <import> and <config-replacers> tags are not supported

Reviewer: ```suggestion
* <import> and <config-replacers> tags
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021

> * Variables in .xml configuration are not supported

Reviewer: ```suggestion
* Variables in .xml configuration
```

Author: Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021
